### PR TITLE
Storing JSON file with Kved data to DB

### PR DESCRIPTION
**Pull request by IegorSoboliev**
Added JSON storing to DB

### CODE REVIEW

File: ratu/migrations/0034_auto_20200421_1325.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Generated by Django 2.0.9 on 2020-04-21 13:25
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('ratu', '0033_merge_20200421_0929'),
```
Comment (criticus) [inline ratu/migrations/0034_auto_20200421_1325.py:10]: @Iva-khar is this correct one?

Comment (Iva-khar) [reply ratu/migrations/0034_auto_20200421_1325.py:10]: yes

**Comment by criticus** [comment]
can you please change the name of this PR to something more informative?

File: ratu/models/kzed_models.py [hunk 2]
```diff
@@ -1,26 +1,44 @@
 from django.db import models
 
 class Section(models.Model):
+    EMPTY_FIELD = 'empty field'
```
Comment (criticus) [inline ratu/models/kzed_models.py:4]: where do we use this?

File: ratu/models/kzed_models.py [hunk 3]
```diff
@@ -1,26 +1,44 @@
...
     code = models.CharField(max_length=3, unique=True)
     name = models.CharField(max_length=150)
 
 
+    def __str__(self):
+            return self.name
+
+
 class Division(models.Model):
+    EMPTY_FIELD = 'empty field'
```
Comment (criticus) [inline ratu/models/kzed_models.py:14]: same here. is it needed?

File: ratu/models/kzed_models.py [hunk 4]
```diff
@@ -1,26 +1,44 @@
...
     code = models.CharField(max_length=5, unique=True)
     name = models.CharField(max_length=150)
 
 
+    def __str__(self):
+            return self.name
+
+
 class Group(models.Model):
+    EMPTY_FIELD = 'empty field'
```
Comment (criticus) [inline ratu/models/kzed_models.py:25]: and here?

File: ratu/models/kzed_models.py [hunk 5]
```diff
@@ -1,26 +1,44 @@
...
-#     division = models.ForeignKey(Division, on_delete=models.CASCADE)
-#     group = models.ForeignKey(Group, on_delete=models.CASCADE)
-#     code = models.CharField(max_length=10, unique=True)
-#     name = models.CharField(max_length=100)
+    def __str__(self):
+            return self.name
+
+
+class Kzed(models.Model):
+    EMPTY_FIELD = 'empty field'
```
Comment (criticus) [inline ratu/models/kzed_models.py:36]: and one more

File: ratu/models/ruo_models.py [hunk 6]
```diff
@@ -1,4 +1,5 @@
 from django.db import models
+from ratu.models.kzed_models import Kzed
```
Comment (criticus) [inline ratu/models/ruo_models.py:2]: import but no usage?

File: ratu/models/kzed_models.py (hunk 2 above)
Comment (IegorSoboliev) [reply ratu/models/kzed_models.py:4]: for default kved

File: ratu/models/kzed_models.py (hunk 3 above)
Comment (IegorSoboliev) [reply ratu/models/kzed_models.py:14]: I have thought we need that for default kved

File: ratu/models/kzed_models.py (hunk 4 above)
Comment (IegorSoboliev) [reply ratu/models/kzed_models.py:25]: the same reason as above

File: ratu/models/kzed_models.py (hunk 5 above)
Comment (IegorSoboliev) [reply ratu/models/kzed_models.py:36]: yes)

File: ratu/models/ruo_models.py (hunk 6 above)
Comment (IegorSoboliev) [reply ratu/models/ruo_models.py:2]: need this for future integration  Kved and Ruo models

**Pull request merged**